tests/js/spec/components/dropdownLink.spec.jsx

```diff
@@ -131,7 +131,14 @@ describe('DropdownLink', function () {
       });
 
       it('does not close when document is clicked', function () {
-        $(document).click();
+        const event = document.createEvent('HTMLEvents');
+        event.initEvent('click', false, true);
+        document.dispatchEvent(event);
+
+        const evt = document.createEvent('HTMLEvents');
+        evt.initEvent('click', false, true);
+        document.body.dispatchEvent(evt);
```

Comment:
these two blocks of code are very similar, except for `evt` and `event`. Should we remove one of them?

Reply:
oops sorry, didn't push my fix 🙃